packages/kotti-ui/source/kotti-field-number/KtFieldNumber.vue

```diff
@@ -411,7 +411,8 @@ export default defineComponent<KottiFieldNumber.PropsInternal>({
 			}
 
 			&--is-disabled {
```

Comment:
Also the computed `inputProps` isn't adding the `--is-disabled` class, so this isn't doing anything.
@carsoli fyi

Reply:
This is actually doing something when the whole field is `disabled`, I put the logic in the wrong class.
Now is updated.

Reply:
Anything inside `kt-field-number__middle__input--is-disabled` is not doing anything because the computed `inputProps` is not adding the `--is-disabled` class.

However, this is out of scope so it can be ignored.

Reply:
Resolving. This unused style is now remove in https://github.com/3YOURMIND/kotti/pull/783